### DISCUSSION TITLE
Pipeline Time in Graphana?

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#8607] [Unsupported Versions] [answered]
Does someone know what the Pipline Time graph is showing?

**Comment by isaacgolding** [comment]
The pipeline has a lot of information about how data is being processed.  Can you be a bit more specific in your question?

**Reply by OzzyKampha** [reply]
When i choose the pipline dashboard. It has one graph with the title "Pipline Time". It has days as y-axes. My question is; what is this spesific graph showing?

**Comment by OzzyKampha** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/issues/8369
